src/mlpack/methods/preprocess/preprocess_describe_main.cpp

```diff
@@ -223,19 +223,22 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
     const double fStd = arma::stddev(feature, population);
 
     // Print statistics of the given dimension.
-    Log::Info << boost::format(numberFormat)
-        % dim
-        % arma::var(feature, population)
-        % fMean
-        % fStd
-        % arma::median(feature)
-        % fMin
-        % fMax
-        % (fMax - fMin) // range
-        % Skewness(feature, fStd, fMean, population)
-        % Kurtosis(feature, fStd, fMean, population)
-        % StandardError(feature.n_elem, fStd)
-        << endl;
+    Log::Info << setw(width) << setprecision(precision) << numberFormat <<
```

Comment:
Are you sure you have to set the precision each time?

Reply:
Done